src/main/java/uk/gov/dvsa/enums/CertificateTemplates.java

```diff
@@ -33,7 +33,9 @@ public enum CertificateTemplates {
 
     RWT_DATA("CommercialVehicles/RWT_DATA"),
 
-    INSPECTION_CHECKLIST("MOT/Inspection_Checklist");
+    INSPECTION_CHECKLIST("MOT/Inspection_Checklist"),
+
+    IVA30_FAIL("CommercialVehicles/IVA30_Fail");
```

Comment:
Not sure this needs the _FAIL as an IVA30 is a fail.  the HGV/PSV template names don't include the pass/fail, just the document name that corresponds to pass/fail

Reply:
Thanks for the feedback, implemented :)